src/bootstrap/bin/rustc.rs

```diff
@@ -315,6 +315,8 @@ fn main() {
         cmd.arg("-Z").arg("force-unstable-if-unmarked");
     }
 
+    cmd.arg("-Z").arg("binary-dep-depinfo");
```

Comment:
Could this include a comment indicating why it's doing this?

Reply:
Also, is it possible to use the Cargo feature for this instead of manually editing the `rustc.rs` wrapper here?

Reply:
Ah, sure, I didn't know we had threaded this out from Cargo, I'll definitely prefer to do that!